xchange-okcoin/src/main/java/com/xeiam/xchange/okcoin/service/polling/marketdata/OkCoinMarketDataService.java

```diff
@@ -30,7 +30,11 @@ public Ticker getTicker(CurrencyPair currencyPair, Object... args) throws IOExce
 
   @Override
   public OrderBook getOrderBook(CurrencyPair currencyPair, Object... args) throws IOException {
-    return OkCoinAdapters.adaptOrderBook(getDepth(currencyPair), currencyPair);
+    if (args.length != 0)
+      return OkCoinAdapters.adaptOrderBook(getDepth(currencyPair, (Integer) args[0]), currencyPair);
+    else
+      return OkCoinAdapters.adaptOrderBook(getDepth(currencyPair, 200), currencyPair);
```

Comment:
I would add comment saying that if 'size' is not specified, the maximum depth of 200 is returned, so it is clear that this change does not impact old code which does not specify size